webpack.config.babel.js

```diff
@@ -208,17 +209,17 @@ const configDist = {
     }),
     ... PRODUCTION_BUILD ?
       [
-        new webpack.optimize.UglifyJsPlugin({
-          beautify: false,
+        new UglifyJSPlugin({
           sourceMap: true,
-          compress: {
-            pure_getters: true,
-            unsafe: true,
-            unsafe_comps: true,
-            screw_ie8: true,
-            warnings: false,
-            unused: true,
-            dead_code: true
```

Comment:
any ideas if the non included options are necessary? such as `dead_code`, `unused`, `screw_ie8`?

Reply:
As above I have removed any flag where we are using the default (see https://github.com/mishoo/UglifyJS2) `screw_ie8` was renamed to `ie8`